modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java

```diff
@@ -720,7 +723,8 @@ public Comment postByExternalReferenceCodeComment(
 
 		return CommentUtil.toComment(
 			_commentManager.addEntityComment(
-				comment.getExternalReferenceCode(), objectEntry.getScopeId(),
+				comment.getExternalReferenceCode(),
+				_getGroupId(objectEntry.getScopeId(), objectEntry.getId()),
```

Comment:
I think you can call `objectEntry.getNonzeroGroupId()` directly here, so there is not need to create `_getGroupId` method. You can use an approach similar to [ObjectEntryResourceImpl#getPermissionCheckerGroupId](https://github.com/liferay/liferay-portal/blob/6d40e933222a2e5678e30046b6bb04173be7071b/modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/resource/v1_0/ObjectEntryResourceImpl.java#L1290)